aries-site/src/examples/templates/forms/formHelpers.js

```diff
@@ -0,0 +1,62 @@
+export const emailMask = [
+  {
+    regexp: /^[\w\-_.]+$/,
+    placeholder: 'jane.smith',
+  },
+  { fixed: '@' },
+  {
+    regexp: /^[\w]+$/,
+    placeholder: 'hpe',
+  },
+  { fixed: '.' },
+  {
+    regexp: /^[\w]+$/,
+    placeholder: 'com',
+  },
+];
+
+export const emailValidation = [
+  {
+    regexp: new RegExp('[^@ \\t\\r\\n]+@'),
+    message: 'Missing an @?',
+    status: 'info',
+  },
+  {
+    regexp: new RegExp('[^@ \\t\\r\\n]+@[^@ \\t\\r\\n]+\\.[^@ \\t\\r\\n]+'),
+    message: 'Missing an .?',
+    status: 'info',
+  },
+  {
+    regexp: new RegExp('[^@ \\t\\r\\n]+@[^@ \\t\\r\\n]+\\.[^@ \\t\\r\\n]+'),
+    message: "Email address doesn't look quite right",
+    status: 'error',
+  },
+];
+
+export const passwordValidation = [
```

Comment:
This may be more of a design question, but from my perspective, if I'm creating a password and there are multiple criteria that it needs to contain (i.e. Capital letter, number, special character), I'd like to know all of those from the start as opposed to have them slowly revealed as I type something. If I could see the criteria as a list that then validated as I typed, I would feel better about my password. Otherwise, I'm creating my password in a very reactionary manner.

Reply:
Absolutely. There is quite a bit of work to do on form interaction (at form level as well as field level). I will take a look to see what can be done upfront without too much scope creep.

Reply:
It could be done with axe if we loop through all the routes on the site, then every page will be checked for accessibility but I didn't know if that would add too much time to the tests. If you want to just to a test as you develop, you can add WAVE chrome browser extension

Reply:
Maybe some `help` text?

Reply:
Applied `help` text.